### ISSUE TITLE
Allow text selection when input type="text"

### ISSUE THREAD

**Issue by mb-dev** [selectize/selectize.js#256]
Currently it is impossible to use arrow keys + left/right, or even CTRL+A to select the item text. 

For this request - my interest is in single item selection on a input type="text". So selectize becomes a better 'typeahead' with the nice to have 'add <newitem>' prompt. I wonder if this use case can be supported and then the selected item can be inside the hidden input type="text" and thus selectable.

**Comment by joallard** [comment]
Annoying indeed. STR:
1. On http://selectize.github.io/selectize.js/, single item select, choose Tesla
2. Click back on select
3. Hit Ctrl+A.
4. => Text is not selected, expected select-all
5. Hit Shift+Left
6. => Text is not selected, expected select last letter

**Comment by joallard** [comment]
Oops

**Comment by rwcorbett** [comment]
Any idea what event is firing when `ctrl-a` is pressed?

It does not appear to be blur, change, clear, or delete based on logging of those events to console:
```
...
  onDelete: function(d){console.log("selectize onDelete", d);},
  onClear: function(){console.log("selectize onClear");},
  onBlur: function (){console.log("selectize onBlur");},
  onChange: function(){console.log("selectize onChange");}
});
```

**Comment by markhealey** [comment]
Hi, is there an update on this one? Still present in latest AFAIK.

**Comment by MathieuDR** [comment]
I still don't think this is solved. 
Sadly.

One of the first things I noticed. Also clicking the 'tag' won't remove or edit it.

Would love an update.

**Comment by Sebbb** [comment]
Indeed. Ctrl+A works, but removes the entered text. Selecting multiple elements by pressing shift and moving the cursor to the left also does not work and would be a really useful feature.

**Comment by markhealey** [comment]
Reopen, please.
